exercises/outcomes/LT/LT7/template.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<knowl mode="exercise" xmlns="https://spatext.clontz.org" version="0.2">
+    <!-- {{#package}} -->
```

Comment:
I still see these - push a commit with these removed and I can inspect if there's still an issue.

Reply:
Done, sorry about that. I shouldn't try to rush these. Forgot to actually push the commit.